Tests/RSCoreTests/MacroProcessorTests.swift

```diff
@@ -28,8 +28,8 @@ class MacroProcessorTests: XCTestCase {
 		result = try! MacroProcessor.renderedText(withTemplate: template, substitutions: substitutions)
 		XCTAssertEqual(result, expected)
 
-		// Nonexistant key
-		template = "foo [[nonexistant]] bar"
+		// Nonexistent key
+		template = "foo [[nonexistent]] bar"
```

Comment:
Did you test to make sure that this didn't break the test case?

Reply:
No dev environment to test this. Was easier to revert.